dashboard/app/reporting_email.go

```diff
@@ -470,6 +477,7 @@ func incomingMail(c context.Context, r *http.Request) error {
 		incomingBugListEmail(c, bugListInfo, msg)
 		return nil
 	}
+	saveDiscussionEmail(c, bugInfo, msg, emailConfig)
```

Comment:
If we subscribe kernel lists, we need to update the way we extract bugInfo. These emails won't necessary cc syzbot email.